service/topology_coordinator.cc

```diff
@@ -773,35 +771,41 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                 utils::UUID req_uuid = *_topo_sm._topology.global_request_id;
                 std::vector<canonical_mutation> updates;
                 sstring error;
-                size_t unimportant_init_tablet_count = 2; // must be a power of 2
-                locator::tablet_map new_tablet_map{unimportant_init_tablet_count};
+                if (_db.has_keyspace(ks_name)) {
+                    auto& ks = _db.find_keyspace(ks_name);
+                    auto tmptr = get_token_metadata_ptr();
+                    size_t unimportant_init_tablet_count = 2; // must be a power of 2
+                    locator::tablet_map new_tablet_map{unimportant_init_tablet_count};
 
-                for (const auto& table : ks.metadata()->tables()) {
-                    try {
-                        locator::tablet_map old_tablets = tmptr->tablets().get_tablet_map(table->id());
-                        locator::replication_strategy_params params{repl_opts, old_tablets.tablet_count()};
-                        auto new_strategy = locator::abstract_replication_strategy::create_replication_strategy("NetworkTopologyStrategy", params);
-                        new_tablet_map = co_await new_strategy->maybe_as_tablet_aware()->reallocate_tablets(table, tmptr, old_tablets);
-                    } catch (const std::exception& e) {
-                        error = e.what();
-                        rtlogger.error("Couldn't process global_topology_request::keyspace_rf_change, error: {},"
-                                       "desired new ks opts: {}", error, new_ks_props.get_replication_options());
-                        updates.clear(); // remove all tablets mutations ...
-                        break;           // ... and only create mutations deleting the global req
-                    }
+                    for (const auto& table : ks.metadata()->tables()) {
```

Comment:
Please split indentation changes to a separate commit so the PR is easier to review.

Reply:
I left a note in the description that touches on whitespaces subject:
> (I suggest to disable displaying whitespace differences when reviewing this PR).

So now, so that I remember it once and for all, are maintainers, you included, good with just disable whitespaces diff or is that I always should extract the "refactor" part into a separate commit?

Reply:
Interesting, I didn't know about this feature.

I don't know what to decide and don't have a strong opinion. One potential reason for extracting whitespace changes to separate commits is that it improves experience when browsing the `git log` -- which I sometimes do when e.g. debugging a regression, and trying to find where it happened. In theory I could in that case find the PR and there use the whitespace hiding feature, but that's more jumping and I don't know.

@scylladb/scylla-maint any opinions on this matter?

Reply:
I'm not sure about `git log`, but `git blame` has extra options to ignore whitespace changes and detect movements of code within/outside of a file, so refactoring the code, i.e. modifying whitespaces or moving it, shouldn't make it that harder to track down a bug.

Reply:
But I don't want to disable whitespace changes when reviewing PRs, making sure whitespace is just right is the single most important duty of a maintainer!
On a serious note, I don't think ignoring whitespace while reviewing is a good idea. It is a good idea when using git blame.
So let's keep splitting reformatting into separate patches, as before.

Reply:
I second Botond's suggestion -- though toggling display of whitespaces differences is possible, it doesn't make review easier

Reply:
Thanks, I split the changes.